layouts/article.html

```diff
@@ -15,6 +15,12 @@
         <div class="markdown-body document-content">
           <% section = lookup_section('toc', @item[:title]) %>
           <h1><%= section << ' ' << @item[:title] if section %></h1>
+          <% improve_this_page_url = "https://github.com/atom/flight-manual.atom.io/edit/master/content" + @item.identifier %>
+          <h1 class="improve">
```

Comment:
@gjtorikian Based on my understanding of ruby and nanoc, this should do the trick. But, I'm open to suggestions on how to style this. For now, I applied (but didn't implement) an `improve` class to the element, and I just went with the default `<h1>` styling to match the page title.

<img width="744" alt="2016-03-24 - atom flight manual improve this page link 03-24-16 7 53 49 pm" src="https://cloud.githubusercontent.com/assets/330036/14034484/fea4abce-f1f9-11e5-8246-dff1d633696a.png">